src/tools/CMakeLists.txt

```diff
@@ -68,6 +68,34 @@ if(AXOM_ENABLE_QUEST)
             )
 endif()
 
+#------------------------------------------------------------------------------
+# data_collection_util is a utility that generates blueprint-conforming sidre
+# output files from an input mfem mesh
+#------------------------------------------------------------------------------
+if( AXOM_ENABLE_SIDRE AND HDF5_FOUND AND MFEM_FOUND AND AXOM_ENABLE_MFEM_SIDRE_DATACOLLECTION )
```

Comment:
Does HDF5_FOUND need to be checked here?

Reply:
It's necessary for writing out the files. 

- [ ] Make HDF5 optional and change the protocol when HDF5's not available

Reply:
I wasn't actually setting the protocol -- just using the default.

It looks like the MFEMSIdreDataCollection should be updated to deal with cases where HDF5s' not enabled. I'll create a ticket for that so we can handle it later.

Reply:
Update: https://github.com/LLNL/axom/issues/623